src/text/text.rs

```diff
@@ -441,6 +441,20 @@ impl<'a> Text<'a> {
     pub fn iter_mut(&mut self) -> std::slice::IterMut<Line<'a>> {
         self.lines.iter_mut()
     }
+
+    /// Adds a line to the text.
+    pub fn push_line(&mut self, line: Line<'a>) {
+        self.lines.push(line);
+    }
+
+    /// Adds a span to the last line of the text.
+    pub fn push_span(&mut self, span: Span<'a>) {
```

Comment:
What about `Into<Line/Span>`?

```suggestion
    pub fn push_span<S: Into<Span<'a>>(&mut self, span: S) {
```

Then you could just add unstyled String for example.

Reply:
Good idea. I'll make that change later tonight